wasm/jit/mmap.go

```diff
@@ -3,11 +3,22 @@
 
 package jit
 
-import "syscall"
+import (
+	"runtime"
+	"syscall"
+)
 
 // mmapCodeSegment copies the code into the executable region and returns the byte slice of the region.
 // See https://man7.org/linux/man-pages/man2/mmap.2.html for mmap API and flags.
 func mmapCodeSegment(code []byte) ([]byte, error) {
+	if runtime.GOARCH == "amd64" {
+		return mmapCodeSegmentAMD64(code)
+	} else {
+		return mmapCodeSegmentARM64(code)
```

Comment:
I will add the diff..

Reply:
https://github.com/tetratelabs/wazero/pull/201/commits/46fa346052306ba4f2a6c2a988d54fc8ee55072f commented